content/docs/start/index.md

```diff
@@ -1,3 +1,7 @@
+---
+description: 'Get started with DVC!'
```

Comment:
Let's try this:
```suggestion
title: 'Get Started | DVC'
description: 'Get started with DVC!'
```

Reply:
I believe this will end up as "Get Started | DVC | DVC", as the bit is auto-added in the SEO component's helmet setup.

Reply:
We'll know soon... ⌛ (I'll report back.)